main.py

```diff
@@ -0,0 +1,16 @@
+import sys
+from parser import Parser
+from router import Router
+
+def main():
+    parser = Parser()
```

Comment:
i think we consciously put it into a class, so the logic can be put into that class should we add more optional flags to our code later.